vitrina/orgs/views.py

```diff
@@ -54,3 +65,58 @@ def get_context_data(self, **kwargs):
         organization: Organization = self.object
         context_data['members'] = organization.representative_set.all()
         return context_data
+
+
+class RepresentativeCreateView(LoginRequiredMixin, PermissionRequiredMixin, CreateView):
+    model = Representative
+    form_class = RepresentativeCreateForm
+    template_name = 'base_form.html'
+
+    def has_permission(self):
+        return self.request.user.role == Representative.COORDINATOR
+
+    def form_valid(self, form):
+        self.object: Representative = form.save(commit=False)
+        organization_id = self.kwargs.get('organization_id')
+        organization = get_object_or_404(Organization, pk=organization_id)
+        self.object.organization = organization
+        try:
+            user = User.objects.get(email=self.object.email)
+            user.organization = organization
```

Comment:
One user can be a member of multiple organizations. This line should be deleted.